test/unittests/parser_test.cpp

```diff
@@ -45,8 +45,14 @@ TEST(parser, valtype)
     EXPECT_EQ(std::get<0>(parse<ValType>(&b)), ValType::i64);
     b = 0x7f;
     EXPECT_EQ(std::get<0>(parse<ValType>(&b)), ValType::i32);
+    b = 0x7c;
+    EXPECT_THROW_MESSAGE(
+        std::get<0>(parse<ValType>(&b)), parser_error, "unsupported valtype (floating point)");
```

Comment:
These don't need `std::get<0>`

Reply:
How would you rewrite them?

Reply:
EXPECT_THROW_MESSAGE(parse<ValType>(&b), parser_error, "unsupported valtype (floating point)");

Reply:
Oh right, for the failing case :)

I thought you also meant for the success cases.

Reply:
Fixed.